app/views/projects/show.html.erb

```diff
@@ -19,6 +19,158 @@
         it is not officially supported.
       </div>
     <% end %>
+
+    <h3 role="button" data-toggle="collapse" class="result-title collapsed" id="project-progress" href="#project-progress-container" aria-expanded="false" aria-controls="result-cnt-project_stats">
+    
+  
+    <% unless @project.miga.metadata[:arcgis].nil? %>
```

Comment:
This seems to be here by mistake. Please check

Reply:
The line 26 is for hiding the collapse button if the project has never been run.

Reply:
It seems this `arcgis` check is here by mistake, though. This check is to add a map at the top of a project when properly configured ([see this example](http://microbial-genomes.org/projects/GROS)).

Reply:
Oh, I see. Thanks for the example. I misunderstood this code. I thought this was for hiding the collapse button if there were no result data to show. I will delete it.